src/ft/mod.rs

```diff
@@ -85,6 +89,10 @@ pub struct FreeTypeRasterizer {
     loader: FreeTypeLoader,
     fallback_lists: HashMap<FontKey, FallbackList>,
     device_pixel_ratio: f32,
+
+    /// Rasterizer creation time stamp to delay lazy font config updates
+    /// in `Rasterizer::laod_font`.
+    initialization_time_stamp: Option<Instant>,
```

Comment:
```suggestion
    creation_timestamp: Option<Instant>,
```

I'd suggest creation here since it's shorter, but it should definitely be `timestamp` instead of `time_stamp`.